routes/auth-routes.js

```diff
@@ -22,14 +22,14 @@ router.post('/login', async (ctx) => {
 })
 
 router.post('/register', async (ctx) => {
-    let employee = await db.employee.find({
+    let employee = await db.students.find({
         where: {email_id: ctx.request.body.email_id}
     })
     if (employee) {
```

Comment:
Change this variable and it's initialization name as well. And give priority to `const` over `let`.